pkg/logger/zap.go

```diff
@@ -52,3 +55,8 @@ func VLevelEncoder(l zapcore.Level, enc zapcore.PrimitiveArrayEncoder) {
 
 // NullTimeEncoder skips time serialization.
 func NullTimeEncoder(t time.Time, enc zapcore.PrimitiveArrayEncoder) {}
+
+// NewAtomicLevelAt returns an appropriate zap.AtomicLevel given an integer representing the log level.
+func NewAtomicLevelAt(level int) zap.AtomicLevel {
```

Comment:
does this need to be exported?

Reply:
This was exported mainly so that I could access it from the tests